src/main/java/com/researchspace/dcd/model/DcdRelatedIdentifier.java

```diff
@@ -0,0 +1,22 @@
+package com.researchspace.dcd.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+/**
+ * When creating a new Deposition, identifiers related to the deposition can
+ * be included such as URIs, DOIs, and others
```

Comment:
Has this been tested too?

Reply:
this will be handled by another iteration. For now we are just uploading the file